tools/shared.py

```diff
@@ -333,7 +331,10 @@ def get_clang_version():
   return actual_clang_version
 
 def check_clang_version():
-  expected = '.'.join(map(str, EXPECTED_LLVM_VERSION))
```

Comment:
is EXPECTED_LLVM_VERSION not used anywhere else? i think it might be, maybe in the tests? I could be wrong.